service/userService.js

```diff
@@ -0,0 +1,53 @@
+const bcrypt = require('bcrypt');
+
+const userDao = require('../model/userDao');
+const {
+  pwValidationCheck,
+  emailValidationCheck,
+} = require('../utils/validation-check.js');
+
+const signUp = async (
+  email,
+  password,
+  name,
+  phoneNumber,
+  address,
+  gender,
+  birthDate,
+  points
+) => {
+  pwValidationCheck(password);
+  emailValidationCheck(email);
+
+  if (isExistedUser(email)) {
+    const error = new Error('EMAIL_EXISTS');
+    error.statusCode = 400;
+    throw error;
+  }
+
+  const saltRounds = 5;
```

Comment:
saltRounds 값은 8~12 사이 값을 넣어주세요
```suggestion
  const saltRounds = 10;
```